src/Forms/DoNotBreakAfterListEdit.cs

```diff
@@ -208,16 +205,10 @@ private void listBoxNamesEtc_SelectedIndexChanged(object sender, EventArgs e)
             {
                 NoBreakAfterItem item = _noBreakAfterList[idx];
                 textBoxNoBreakAfter.Text = item.Text;
-                if (item.Regex != null)

```

Comment:
cool :)
I've committed it (along with another bugfix) + correct tab order here: https://github.com/SubtitleEdit/subtitleedit/commit/2877320b76893edb9f523f1efa47359fe03adca7